### ISSUE TITLE
Can't log in to Ubuntu Forum with my SSO account

### ISSUE THREAD

**Issue by Chewie420** [canonical-web-and-design/ubuntu.com#11676]
Hi not sure where to post this but I thought I would try here. I am trying to log into Ubuntu Forums to ask a question.

When I try to log in I get the following error.

That username is already in use or does not meet the administrator's standards. If you are wookiee42042013532424234 and you have forgotten your password, click [here](https://ubuntuforums.org/login.php?do=lostpw).

I have rest password and tried so many random users names but always get the same message.  I thought maybe it was my VPN but I still get it after turning it off.

Is this the right place to ask for help?

**Comment by Chewie420** [comment]
Hi thanks so much I am trying to log into https://ubuntuforums.org/

**Comment by carkod** [comment]
Seems like an account issue, I can't reproduce.

**Comment by anthonydillon** [comment]
Yes, would seem to be an account issue. Could you please file a bug of the forum project, please? https://bugs.launchpad.net/ubuntuforums.org. Closing this one in the ubuntu.com project.

**Comment by Chewie420** [comment]
Anything needed from me?

Sent from my iPhone

On Jun 6, 2022, at 2:25 PM, Anthony Dillon ***@***.***> wrote:

﻿

Closed #11676<https://github.com/canonical-web-and-design/ubuntu.com/issues/11676> as not planned.

**Comment by anthonydillon** [comment]
Yes, could you file the bug on the project liked above to get your account investigated.

**Comment by Chewie420** [comment]
I can’t log in to create the bug fix

Sent from my iPhone

On Jun 6, 2022, at 3:51 PM, Anthony Dillon ***@***.***> wrote:

﻿

Yes, could you file the bug on the project liked above to get your account investigated.
